_projects/libremesh-simplify_and_get_closer_to_openwrt.md

```diff
@@ -0,0 +1,101 @@
+---
+name: ""
```

Comment:
can you please add a title (aka name) and a short description?

Reply:
Done! Thanks for noticing this!